examples/jsm/nodes/accessors/InstanceNode.js

```diff
@@ -85,6 +93,22 @@ class InstanceNode extends Node {
 
 	}
 
+	update( /*frame*/ ) {
+
+		if ( this.buffer !== null && this.instanceMesh.instanceMatrix.version !== this.buffer.version ) {
```

Comment:
This check necessary? `this.buffer !== null` `update()` should be called after setup.

Reply:
Indeed! Updated the PR accordingly 👍